tests/Findologic/Response/Xml21ResponseParserTest.php

```diff
@@ -229,16 +229,19 @@ public function filterResponseProvider(): array
             (new ColorFilterValue('beige', 'beige', $color))
                 ->setColorHexCode('#F5F5DC')
                 ->setMedia(new Media('https://blubbergurken.io/farbfilter/beige.gif'))
+                ->setDisplayType('media')
         );
         $expectedColorFilter->addValue(
             (new ColorFilterValue('blau', 'blau', $color))
                 ->setColorHexCode('#3c6380')
                 ->setMedia(new Media('https://blubbergurken.io/farbfilter/blau.gif'))
+                ->setDisplayType('media')
         );
         $expectedColorFilter->addValue(
             (new ColorFilterValue('braun', 'braun', $color))
                 ->setColorHexCode('#94651e')
                 ->setMedia(new Media('https://blubbergurken.io/farbfilter/braun.gif'))
+                ->setDisplayType('media')
```

Comment:
Shouldn't there be tests for displayType color and none?

Reply:
Definitely. Added one.